src/neuroconv/tools/iterative_write.py

```diff
@@ -0,0 +1,163 @@
+import math
```

Comment:
Can you add this tool module to the api docs?

Reply:
Done.

Reply:
Eh, not working here. Let me double check.

Reply:
OK, here they are:
![image](https://github.com/user-attachments/assets/27fbd18e-e240-43bb-9375-a8ce9596ea26)